Apps/Examples/Examples/Models/Examples.swift

```diff
@@ -138,7 +138,7 @@ struct Examples {
                 type: ViewAnnotationMarkerExample.self),
         Example(title: "View annotations: animation",
                 description: "Animate a view annotation along a route",
-                testTimeout: 60,
+                testTimeout: 120,
```

Comment:
I'm not quite sure that it's a good idea to have such a long animation in our ExampleTests

Reply:
Fair :) I reduced the animation duration instead